src/styles/blocks/nav.scss

```
@@ -0,0 +1,48 @@
+.nav {
+  &__list {
+    display: flex;
+    margin: 0;
+    padding: 0;
+  }
+
+  &__item {
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    margin: 0 0 0 20px;
+    height: 60px;
+
+    text-transform: uppercase;
+    cursor: pointer;
+
+    text-decoration: none;
+    color: rgba(6, 11, 53, 1);
+    font-family: Roboto, sans-serif;
+    font-size: 12px ;
+    font-weight: 500;
+
+    &--is-active {
+      position: relative;
+
+      margin: 0;
+
+      color: #00acdc;
+    }
+
+    &--is-active::after {
+      display: block;
+      position: absolute;
+      content: "";
+      bottom: 0;
+      width: 100%;
+
+      border-bottom: 4px solid #00acdc;
+      border-radius: 8px;
+    }
```

CL10 ([SASS] - Make use of SASS nesting - write pseudo-class, pseudo-element selectors inside general selector. As well as media queries.)
```suggestion
    &--is-active {
      position: relative;

      margin: 0;

      color: #00acdc;
  

    &::after {
      display: block;
      position: absolute;
      content: "";
      bottom: 0;
      width: 100%;

      border-bottom: 4px solid #00acdc;
      border-radius: 8px;
    }
   }
```

---

fixed